source/app/store/entities/tracks/data.js

```diff
@@ -25,20 +25,14 @@ export const lensTracks = lensPath(['entities', 'tracks'])
 export const lensTrack = id => lensPath(['entities', 'tracks', id])
 
 const discoveryTrackDescription = `In this track, you can search through multiple documents and find information faster
-and more efficiently, using`+ 
-(ENABLE_KENDRA? ` traditional search based technologies such as Amazon Elasticseach (Amazon ES) as well using state-of-the-art machine learning and natural language enterprise search
-with Amazon Kendra.`: `Amazon Elasticsearch`) +
-`
-
-Once you select a document, you'll be able to interact with Amazon Textract` + 
-(ENABLE_COMPREHEND_MEDICAL? `
-, Amazon Comprehend and Amazon Comprehend Medical` :`
- and Amazon Comprehend`) +
-`
-to experience the power of document digitization and extraction of sensitive information.` +
-(ENABLE_KENDRA?
-`
-
+and more efficiently, using `+ 
+(ENABLE_ELASTICSEARCH? `Amazon Elasticsearch ` : ``) +
+(ENABLE_ELASTICSEARCH && ENABLE_KENDRA? `as well as using ` : ``) +
+(ENABLE_KENDRA? ` state-of-the-art machine learning and natural language enterprise search with Amazon Kendra .`: ``) +
```

Comment:
If `ENABLE_ELASTICSEARCH` is `false` and `ENABLE_KENDRA` is `true`, we end up with 2 contiguous spaces here (`"using " + " state of the art"`)

Reply:
Good catch. Updated